src/UnderlineNav2/UnderlineNav.tsx

```diff
@@ -89,8 +90,10 @@ const overflowEffect = (
       if (index < numberOfListItems) {
         items.push(child)
       } else {
+        const ariaCurrent = child.props['aria-current']
+        const isCurrent = ariaCurrent !== undefined && ariaCurrent !== 'false' && ariaCurrent !== false
```

Comment:
[Minor] This could be simplified to:
```suggestion
        const isCurrent = Boolean(ariaCurrent) && ariaCurrent !== 'false'
```

Reply:
Love it ❤️